internal/cmd/io_npipe.go

```diff
@@ -22,7 +22,7 @@ import (
 
 func init() {
 	// Need to seed for the rng in backoff.NextBackoff()
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
As per here: https://pkg.go.dev/math/rand#Seed, there is no need to set the Seed value anymore. Plus the `rand.New()` is going to return a new random generator with the given seed value. We aren't using that anywhere. I think we can get rid of this `init` method completely?

Reply:
yeah I noticed that this isn't being used anywhere too but wasn't sure if I should be doing cleanup in this PR which was to just upgrade to 1.21. Will update this and push new changes.